test/functional/Java8andUp/src/org/openj9/test/string/StringIndexOfString.java

```diff
@@ -0,0 +1,236 @@
+/*******************************************************************************
+ * Copyright (c) 2001, 2018 IBM Corp. and others
```

Comment:
This is the wrong copyright date. I believe this file is new.

Reply:
Fixed.